File: Week 2/Exercises/scores1.c
#include <stdio.h>
#include <cs50.h>

//Constant
const int TOTAL = 3;

//Prototype
float average (int lenght, int array[]);

int main(void)
{
    int scores[TOTAL];

    for (int i = 0; i < TOTAL; i++)
    {
        scores[i] = get_int("Score:");
    }

    printf ("Average: %f\n", average(TOTAL, scores));
}

float average (int lenght, int array[])
{
    int sum = 0;

    for (int i = 0; i < lenght; i++)
    {
        sum += array[i];
    }
    return sum / lenght;
}

File: Week 6/Exercises/scores1.py
from cs50 import get_int

scores = []
for i in range(3):
    score = get_int("Score: ")
    scores.append(score)
    #or scores.append(get_int("Score: "))

average = sum(scores) / len(scores)
print(f"Average of scores: {average:.2f}")